guides/en/1_getting_started/3_components.md

```diff
@@ -22,11 +22,11 @@ The components system of KAPLAY (the `C` of ECS) is powerful and flexible.
 ## What is a Component?
 
 A component is a piece of code that defines a specific behavior of a game
-object. It usually return a set of **properties** and **methods** that are
+object. It usually defines a set of **properties** and **methods** that are
 attached to the game object.
 
-For example, the [`pos()`](/doc/ctx/pos) component returns the `pos` property
-and the `move()` method:
+For example, the [`pos()`](/doc/ctx/pos) component gives the object a `pos` property
+and a `move()` method:
```

Comment:
It only returns one `move` and `pos`, so the

Reply:
... the what?